exampleCourse/courseInstances/Sp15/assessments/exam02-realTimeGradingDisabled/infoAssessment.json

```
@@ -13,70 +13,82 @@
         {
             "mode": "Public",
             "credit": 100,
-            "timeLimitMin": 5,
+            "timeLimitMin": 30,
```

```suggestion
            "timeLimitMin": 5,
```

I suggest leaving this at 5 minutes, because it's better for instructors who want to experiment with how things work. Everyone is going to need to edit this value anyway for their own exam.